kernel/vm/vmm.c

```diff
@@ -370,10 +370,17 @@ status_t vmm_alloc_physical(vmm_aspace_t *aspace, const char *name, size_t size,
 
     /* map all of the pages */
     int err = arch_mmu_map(&aspace->arch_aspace, r->base, paddr, size / PAGE_SIZE, arch_mmu_flags);
-    LTRACEF("arch_mmu_map returns %d\n", err);
+    if (err) {
```

Comment:
These error catching routines here and below look good, can you split them into a separate commit so they can be taken independent of the meat of the change?